hack/release/generate-plugin-manifest.sh

```diff
@@ -18,18 +18,27 @@ PLUGIN_NAME=kubectl-datadog
 OUTPUT_FOLDER=$GIT_ROOT/dist
 TARBALL_NAME="$PLUGIN_NAME_$VERSION.tar.gz"
 
-DARWIN_AMD64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/datadog-operator_${VERSION}_checksums.txt  | grep "darwin_amd64" | awk '{print $1}')
-WINDOWS_AMD64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/datadog-operator_${VERSION}_checksums.txt  | grep "windows_amd64" | awk '{print $1}')
-LINUX_AMD64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/datadog-operator_${VERSION}_checksums.txt  | grep "linux_amd64" | awk '{print $1}')
+DARWIN_AMD64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/checksums.txt  | grep "darwin_amd64" | awk '{print $1}')
+WINDOWS_AMD64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/checksums.txt  | grep "windows_amd64" | awk '{print $1}')
+LINUX_AMD64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/checksums.txt  | grep "linux_amd64" | awk '{print $1}')
+DARWIN_ARM64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/checksums.txt  | grep "darwin_arm64" | awk '{print $1}')
+WINDOWS_ARM64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/checksums.txt  | grep "windows_arm64" | awk '{print $1}')
+LINUX_ARM64=$(grep $PLUGIN_NAME $OUTPUT_FOLDER/checksums.txt  | grep "linux_arm64" | awk '{print $1}')
```

Comment:
you want to ditch `${VERSION}` because you don't expect that the bins will be built on the same machines?

Reply:
it is because the version is note really useful in this context. 
and after we will need to [merge several chechsums.txt files in once](https://github.com/DataDog/datadog-operator/blob/204dffc11f6c404656069bb957f48fee477224dc/.github/workflows/release.yaml#L150-L159), so it was easier to have a "static" file name